Dbscan gui fix

zacsimile opened the pull request:
The real fix for #199

zacsimile committed:
Custom traits editors documentation

zacsimile committed:
Custom view for DBSCANClustering to fix display issues

zacsimile committed:
Remove debug statements

David-Baddeley commented on PYME/recipes/localisations.py:
```diff
@@ -426,6 +426,25 @@ def execute(self, namespace):
 
+    @property
+    def default_view(self):
+        import wx
```
What is this doing here?

David-Baddeley replied:
IE we shouldn't need to test for wx.App

zacsimile replied:
Can delete. Was following the example you told me to use as a template, `recipes.tablefilters`, line 71.

David-Baddeley replied:
sorry, the ExtractTableChannel one would probably have been a better example. Have just made a few tweaks (and about to make a couple more).

zacsimile committed:
Remove wx check

David-Baddeley committed:
Quick view changes

- default view has separators between inputs, parameters, outputs
- only have an OK button, as traits don't roll-back when you click cancel (i.e. clicking cancel is the same as clicking OK)
- added pipeline view

David-Baddeley committed:
fix _pipeline_view in DBScan

David-Baddeley commented:
PS - should really document what the pipeline_view is about - if not immediately obvious, this is what gets shown in the pipeline view on the left in VisGUI.
